"use strict";(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[47],{5249:(e,t,a)=>{function s(e){let{src:t,width:a,quality:s}=e;return"https://res.cloudinary.com/demo/image/upload/".concat(["f_auto","c_limit","w_".concat(a),"q_".concat(s||"auto")].join(",")).concat(t)}a.r(t),a.d(t,{default:()=>s})},1599:(e,t,a)=>{a.r(t),a.d(t,{default:()=>$});var s=a(4848),i=a(6540),n=a(8154),l=a(4042),o=a.n(l);let r={src:"/_next/static/media/about1.b8798cb6.png"},c={src:"/_next/static/media/aboutppl.38fa5c10.png"},d={src:"/_next/static/media/solution.e7406f8b.png"},m=[{nameIco:"fa-brands fa-github",link:"https://github.com/apliko-xyz/PyTDC"},{nameIco:"fa-brands fa-discord",link:"https://discord.gg/6zdHZ3tT"},{nameIco:"fa-solid fa-face-smile",link:"https://huggingface.co/apliko"},{nameIco:"fa-solid fa-file-pdf",link:"https://openreview.net/forum?id=HV8vZDDoYc"}],h=[{title:"tdc_ml.scDTI",content:"Single-cell drug-target nomination (identification). The goal is to train a model for predicting the probability that a protein is a candidate therapeutic target in a specific cell type.",link:"https://tdcommons.ai/multi_pred_tasks/scdti"},{title:"tdc_ml.PerturbOutcome",content:"We define a task for predicting responses in gene expression of single cells to chemical and genetic perturbations, aiming to measure model generalization across cell lines and perturbation types.",link:"https://tdcommons.ai/multi_pred_tasks/counterfactual"},{title:"tdc_ml.ProteinPeptide",content:"Capturing the dynamic nature of protein-peptide interactions requires advanced sampling techniques and computational resources, making the prediction of binding affinities more complex (Antes et al., 2014). We provide benchmarks integrating newly discovered peptides to test ML models on their ability to generalize to cutting-edge peptidomimetics.",link:"https://tdcommons.ai/multi_pred_tasks/proteinpeptide"},{title:"tdc_ml.TCREpitope",content:" T-cells are an integral part of the adaptive immune system, whose survival, proliferation, activation and function are all governed by the interaction of their T-cell receptor (TCR) with immunogenic peptides (epitopes). This new task aims to predict the binding affinity given a pair of TCR sequence and epitope sequence. The models are, at very least, expected to generalize to unseen TCRs. But the main challenge of this dataset is to generalize to samples where both epitope and TCR are unseen.",link:"https://tdcommons.ai/multi_pred_tasks/tcrepitope"},{title:"tdc_ml.TrialOutcome",content:"Clinical trial outcome prediction is a machine learning task that aims to forecast the outcome of clinical trials, such as the approval rate of a drug or treatment. It utilizes various clinical trial features, including the drug's molecular structure, disease code representing the medical condition, and eligibility criteria that specify participant selection criteria. Machine learning models for clinical trial outcome prediction are expected to demonstrate robust generalization to novel drug molecular structures and rare diseases. The ability to generalize well to diverse and evolving conditions is crucial for the models to be adaptable and effectively contribute to the field of clinical trials.",link:"https://tdcommons.ai/multi_pred_tasks/trialoutcome"},{title:"tdc_ml.SBDD",content:"Structure-based Drug Design is to generate diverse, novel molecules that have high binding affinity to protein pockets (3D structures) and desirable chemical properties. These properties are measured by oracle functions. A machine learning task first learns the molecular characteristics given specific protein pockets from a large set of protein-ligand pair data. Then, from the learned conditional distribution, we can sample novel candidates.",link:"https://tdcommons.ai/multi_pred_tasks/sbdd"}],u=[{timelineClass:"block block-right mt-30",title:"tdc_ml.TCREpitope Introduced",date_from:"Feb 19th, 2022",date_to:"June 19th, 2024",content:"TDC has a new task on TCR-Epitope Binding prediction (Thanks to Anna and Jannis)! We enhanced the task by consolidating TCHard and PanPep datasets and further ran benchmarks for numerous SoTA models in our publications."},{timelineClass:"block block-left",title:"Introduction of PyTDC Resource model and tdc_ml.PrimeKG",date_from:"July 27,2022",date_to:"Sep 24th, 2024",content:"Piloted by the introduction of the PrimeKG API to TDC, the TDC Resource module became the data model for the later introduction of TDC's API-first architecture. Enhancements have been implemented for the PrimeKG API since."},{timelineClass:"block block-right mt-30",title:"tdc_ml.SBDD introduced",date_from:"November 3rd, 2022",date_to:"June 19th, 2024",content:"TDC has a new task on structure-based drug design SBDD with four datasets PDBBind, DUD-E, scPDB. To support evaluation of SBDD tasks, we also include two evaluation metrics (RMSD, Kabsch-RMSD) that compare distances between two structures."},{timelineClass:"block block-left mt-30",title:"TDC Huggingface Model Hub released using DeepPurpose framework",date_from:"April 17th, 2023",date_to:"June 19th, 2024",content:"We're excited to announce the release of a new interface tdc_hf_interface that allows users to easily access and leverage pre-trained models hosted at HuggingFace on TDC datasets and tasks. In this first batch, we've released nine pre-trained models from DeepPurpose that cover three popular ADMET datasets in the Commons."},{timelineClass:"block block-right mt-30",title:"tdc_ml.TrialOutcome introduced",date_from:"July 10th, 2023",date_to:"June 19th 2024",content:"TDC has a new exciting task on clinical trial outcome prediction (Thanks to Tianfan)! The release features the TOP dataset. TOP consists of 17,538 clinical trials with 13,880 small-molecule drugs and 5,335 diseases."},{timelineClass:"block block-left mt-30",title:"tdc_ml.ProteinPeptide and API-first architecture introduced",date_from:"March 22nd, 2024",date_to:"June 19th 2024",content:"TDC introduced the new protein-peptide binding affinity task along with the Ye X et al. dataset (thanks to Joseph Brown!). Affinity selection-mass spectrometry data of discovered ligands against single biomolecular targets (MDM2, ACE2, 12ca5) from the Pentelute Lab of MIT This dataset contains affinity selection-mass spectrometry data of discovered ligands against single biomolecular targets. Furthermore, the architecture of TDC was redesigned using the model-view-controller design pattern to enable multimodal dataviews, with this dataset serving as the pilot API-first dataset."},{timelineClass:"block block-right mt-30",title:"tdc_ml.PerturbOutcome introduced",date_from:"April 5th, 2024",date_to:"June 19th 2024",content:"TDC introduced the new single-cell genetic and chemical perturbation response prediction tasks along with the scPerturb datasets."},{timelineClass:"block block-left mt-30",title:"PyTDC Multimodal Single-cell Retrieval API introduced",date_from:"April 18th, 2024",date_to:"June 19th 2024",content:"PyTDC introduces over 1,000 multimodal datasets, spanning approximately 85 million cells and pre-calculated foundation model embeddings from 5 state-of-the-art single-cell models via CZ CELLxGENE Census. The resource is integrated into the TDC API-first dataset model to enable retrieval of multimodal dataviews for diverse single-cell therapeutic AI tasks."},{timelineClass:"block block-right mt-30",title:"tdc_ml.scDTI introduced",date_from:"April 26th, 2024",date_to:"June 19th 2024",content:"PyTDC introduced the first-of-its-kind single-cell drug target nomination task. The release features a high-quality curated dataset from OpenTargets (thanks Michelle Li!)."},{timelineClass:"block block-left mt-30",title:"Alpha release of PyTDC, along with single-cell therapeutics AI benchmarks, is presented to MoML2024 at Mila",date_from:"June 19th, 2024",date_to:"June 22nd 2024",content:"PyTDC was released along with its pre-print in biorxiv and spotlight accepted to the Molecular Machine Learning Conference at Mila. Our poster featured single-cell therapeutics AI tasks and benchmarks, the API-first dataset, and the multimodal single-cell retrieval API."},{timelineClass:"block block-right mt-30",title:"PyTDC Model Server is alpha released",date_from:"October 19th, 2024",date_to:"November 11th 2024",content:"PyTDC released the alpha version of the Model Server, open source inference serving software that streamlines AI inferencing for single-cell foundation models across modalities, piloted by an inference pipeline for Geneformer. It is a first-of-its-kind release."},{timelineClass:"block block-left mt-30",title:"PyTDC is published and presented at NeurIPS and other venues",date_from:"November 14th, 2024",date_to:"December 15th 2024",content:'The PyTDC manuscript, "Signals in the Cells: Multimodal and Contextualized Machine Learning Foundations for Therapeutics.", was spotlight at NeurIPS AIDrugX 2024. PyTDC was presented at the Western Bioinformatics Seminar Series and a new poster was presented at AIDrugX. The new poster introduced the PyTDC model server for the first time.'},{timelineClass:"block block-right mt-30",title:"PyTDC beta release",date_from:"TBD",date_to:"TBD",content:"Follow us on Github, Huggingface, and Discord for updates on the beta release of PyTDC, which will feature an exhaustive suite of models among other exciting features. We are working hard to make it available soon!"}],p=[{title:"ICML Paper",link:"https://openreview.net/forum?id=HV8vZDDoYc"},{title:"NeurIPS'24 AIDrugX Spotlight Paper",link:"https://openreview.net/pdf?id=kL8dlYp6IM"},{title:"NeurIPS'24 AIDrugX Poster",link:"https://drive.google.com/file/d/1plypydZCaegbgxyCl-xehFxSgwX6e8So/view?usp=sharing"},{title:"Western Bioinformatics Seminar Slides",link:"https://drive.google.com/file/d/197Zay3tu0F-MJw_orf0QRWyyhkwN6lqC/view?usp=sharing"},{title:"MoML'24 Conference @ Mila Poster",link:"https://drive.google.com/file/d/1LYdITeFY5iX07zyXPGVEjMpYjuHMrneS/view?usp=sharing"}],g=[{img:{src:"/_next/static/media/1.560090ca.png",height:500,width:500,blurDataURL:"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAICAMAAADz0U65AAAAKlBMVEUJEBUAAABMaXEdHSAIIS4WEhEBEBqIZ1lbRz5BMCm4jHoAFiQEHCskIiTUc3GMAAAADHRSTlMjAQCAfOPk+/v29vK9GIVjAAAACXBIWXMAAA7EAAAOxAGVKw4bAAAAM3RFWHRDb21tZW50AHhyOmQ6REFGRXRtUW1xV0U6NSxqOjI5NDg3Nzk2NjE4LHQ6MjIwNjI2MTQngCN5AAAANUlEQVR4nE2LOQ7AMAyAMM7Vyvn/d6tk6oDEABjhgZBMDGTMNRCz3llNbLWeK/S9O//47IYfH0sA4npD+UEAAAAASUVORK5CYII=",blurWidth:8,blurHeight:8},title:"Alex V",text:"Founder, CEO"}],f=[{img:{src:"/_next/static/media/1.b7ab58d7.png",height:174,width:448,blurDataURL:"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAADCAMAAACZFr56AAAAJFBMVEUaLEE7T2klRW4iNllIVGwcMkooTHlNX3xdcYkhL1M2SliJk6Frdeu0AAAACXBIWXMAABYlAAAWJQFJUiTwAAAAIUlEQVR4nAXBhwEAMAjDMIcN/f/fSrjWrcrQi+gewQFkfgWCAGAh68lKAAAAAElFTkSuQmCC",blurWidth:8,blurHeight:3},link:"https://aplusventures.io/"},{img:{src:"/_next/static/media/community.3b3a99ba.png",height:500,width:500,blurDataURL:"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAICAMAAADz0U65AAAAJ1BMVEUAAAAHCAoUEhchISMzLS1EQEYPDQ8dIjMDFRUkK0A+WFRMPDRAaGk3uoCLAAAACXBIWXMAAA7EAAAOxAGVKw4bAAAAM3RFWHRDb21tZW50AHhyOmQ6REFGRllkUGp5UlU6NyxqOjI5OTk2NDY5MjA2LHQ6MjIwNzAzMTcNHj0YAAAANUlEQVR4nCWISQ7AIAzEZkug0P+/t6L4YFkGIJE4jGfoD9r3sOoGkmNHlOZEyspeL2Q2u/UBEXsAqIG2aGMAAAAASUVORK5CYII=",blurWidth:8,blurHeight:8},link:"https://discord.gg/ZJ67MtXag4"}],x=e=>{"undefined"!=typeof document&&(document.title=e)},j=()=>{if("undefined"!=typeof document){let e=document.querySelector(".navbarToggler");e&&(e.classList.contains("active")?e.classList.remove("active"):e.classList.add("active"),A())}},A=()=>{if("undefined"!=typeof document){let e=document.querySelector(".navbarToggler"),t=document.querySelector(".classy-menu");e&&t&&(e.classList.contains("active")?t.classList.add("menu-on"):t.classList.remove("menu-on"))}},b=()=>{window.scrollTo({behavior:"smooth"})};var v=a(9965),k=a.n(v);let y={src:"/_next/static/media/logo.195faece.png"},N=()=>((0,i.useEffect)(()=>{!function(){if("undefined"!=typeof document){let e=document.querySelector(".classy-nav-container");e&&window.addEventListener("scroll",()=>{window.pageYOffset>86?e.classList.add("sticky"):e.classList.remove("sticky")})}}()},[]),(0,i.useEffect)(()=>{A()},[]),(0,i.useEffect)(()=>{b()},[]),(0,s.jsx)(s.Fragment,{children:(0,s.jsx)("header",{className:"header-area wow fadeInDown","data-wow-delay":"0.2s",children:(0,s.jsx)("div",{className:"classy-nav-container breakpoint-off",children:(0,s.jsx)("div",{className:"container",children:(0,s.jsxs)("nav",{className:"classy-navbar justify-content-between",id:"dreamNav",children:[(0,s.jsx)(k(),{src:y.src,alt:"logo",layout:"fill"}),(0,s.jsx)("div",{className:"classy-navbar-toggler",children:(0,s.jsxs)("span",{className:"navbarToggler",onClick:j,children:[(0,s.jsx)("span",{}),(0,s.jsx)("span",{}),(0,s.jsx)("span",{})]})}),(0,s.jsxs)("div",{className:"classy-menu",children:[(0,s.jsx)("div",{className:"classycloseIcon",children:(0,s.jsxs)("div",{className:"cross-wrap",onClick:j,children:[(0,s.jsx)("span",{className:"top"}),(0,s.jsx)("span",{className:"bottom"})]})}),(0,s.jsxs)("div",{className:"classynav",children:[(0,s.jsxs)("ul",{id:"nav",children:[(0,s.jsx)("li",{children:(0,s.jsx)("a",{onClick:b,href:"#home",className:"login-btn",style:{borderColor:"#FFEFC2",textAlign:"center",alignItems:"center"},children:"Home"})}),(0,s.jsx)("li",{children:(0,s.jsx)("a",{onClick:b,href:"#about",className:"login-btn",style:{borderColor:"#FFEFC2",textAlign:"center",alignItems:"center"},children:"About"})}),(0,s.jsx)("li",{children:(0,s.jsx)("a",{onClick:b,href:"#services",className:"login-btn",style:{borderColor:"#FFEFC2",textAlign:"center",alignItems:"center"},children:"Features"})}),(0,s.jsx)("li",{children:(0,s.jsx)("a",{onClick:b,href:"#team",className:"login-btn",style:{borderColor:"#FFEFC2",textAlign:"center",alignItems:"center"},children:"Team"})})]}),(0,s.jsx)("a",{href:"/home",className:"btn login-btn ml-50",style:{borderColor:"#FFDF00"},children:"Homepage"})]})]})]})})})})})),w=()=>(0,s.jsx)("span",{}),C=()=>(0,s.jsx)("div",{className:"col-12 col-lg-6",children:(0,s.jsxs)("div",{className:"who-we-contant",children:[(0,s.jsxs)("div",{className:"dream-dots","data-aos":"fade-up",children:[(0,s.jsx)(w,{}),(0,s.jsx)(w,{}),(0,s.jsx)(w,{}),(0,s.jsx)(w,{}),(0,s.jsx)(w,{}),(0,s.jsx)(w,{}),(0,s.jsx)(w,{})]}),(0,s.jsx)("h4",{"data-aos":"fade-up",children:"PyTDC API-first Architecture"}),(0,s.jsx)("p",{"data-aos":"fade-up",children:'A collection of multimodal continually updated heterogeneous data sources is unified under the introduced "API-first-dataset" architecture. Inspired by API-first design, this microservice architecture is implemented using the model-view-controller design pattern to enable multimodal data views under a domain-specific-language.'}),(0,s.jsx)("a",{className:"btn dream-btn mt-30",href:"https://colab.research.google.com/drive/1xTgBwKUfP2b8j6Fqh28M2GUp2ScfENMX?usp=sharing",style:{borderColor:"#FFDF00"},children:"CELLxGENE notebook"})]})}),T=e=>{let{img:t}=e;return(0,s.jsx)("div",{className:"col-12 col-lg-6",children:(0,s.jsx)("div",{className:"welcome-meter","data-aos":"fade-up",children:(0,s.jsx)(k(),{src:t,className:"center-block",alt:"Welcome Meter",layout:"fill"})})})},D=e=>{let{img:t}=e;return(0,s.jsx)("section",{className:"about-us-area section-padding-0-100 clearfix",children:(0,s.jsx)("div",{className:"container",children:(0,s.jsxs)("div",{className:"row align-items-center",children:[(0,s.jsx)(C,{}),(0,s.jsx)(T,{img:t})]})})})},I=e=>{let{img:t}=e;return(0,s.jsx)("div",{className:"col-12 col-lg-6","data-aos":"fade-up",children:(0,s.jsxs)("div",{className:"welcome-meter",children:[(0,s.jsx)(k(),{src:t,className:"img-responsive center-block",alt:"",layout:"fill"}),(0,s.jsx)("div",{className:"growing-company text-center mt-30"})]})})},M=()=>(0,s.jsx)("span",{}),_=()=>(0,s.jsx)("div",{className:"col-12 col-lg-6",children:(0,s.jsxs)("div",{className:"who-we-contant",children:[(0,s.jsxs)("div",{className:"dream-dots",children:[(0,s.jsx)(M,{}),(0,s.jsx)(M,{}),(0,s.jsx)(M,{}),(0,s.jsx)(M,{}),(0,s.jsx)(M,{}),(0,s.jsx)(M,{}),(0,s.jsx)(M,{})]}),(0,s.jsx)("h4",{"data-aos":"fade-up",children:"PyTDC Model Server"}),(0,s.jsx)("p",{"data-aos":"fade-up",children:"The platform introduces a model server, which provides unified access to model weights across distributed repositories and standardized inference endpoints. The model server accelerates research workflows by exposing state-of-the-art, research-ready models and training setups for biomedical representation learning models across modalities. A model store retrieval API provides unified access to model weights stored in the Hugging Face Model Hub (https://huggingface.co/apliko), Chan-Zuckerberg CELLxGENE Census fine-tuned models, and TDC storage. The model server also provides access to model classes, tokenizer functions, and inference endpoints supporting PyTorch and Hugging Face Transformers. Extracted embeddings, from either model server inference or pre-computed embedding storage, are ready for downstream use by task-specific benchmarking modules. In this alpha release, the model server exposes SoTA single-cell foundation models (scFMs)."}),(0,s.jsx)("a",{"data-aos":"fade-up",className:"btn dream-btn mt-30",href:"https://huggingface.co/apliko",style:{borderColor:"#FFDF00"},children:"HF Model Hub"})]})}),P=e=>{let{img:t}=e;return(0,s.jsx)("section",{className:"about-us-area section-padding-0-100 clearfix",id:"about",children:(0,s.jsx)("div",{className:"container",children:(0,s.jsxs)("div",{className:"row align-items-center",children:[(0,s.jsx)(I,{img:t}),(0,s.jsx)(_,{})]})})})},E=e=>{let{img:t}=e;return(0,s.jsx)("div",{className:"col-12 col-lg-6","data-aos":"fade-up",children:(0,s.jsxs)("div",{className:"welcome-meter",children:[(0,s.jsx)("a",{children:(0,s.jsx)(k(),{src:t,className:"img-responsive center-block",alt:"",layout:"fill"})}),(0,s.jsx)("div",{className:"growing-company text-center mt-30"})]})})},F=()=>(0,s.jsx)("span",{}),S=()=>(0,s.jsx)("div",{className:"col-12 col-lg-6",children:(0,s.jsxs)("div",{className:"who-we-contant",children:[(0,s.jsxs)("div",{className:"dream-dots",children:[(0,s.jsx)(F,{}),(0,s.jsx)(F,{}),(0,s.jsx)(F,{}),(0,s.jsx)(F,{}),(0,s.jsx)(F,{}),(0,s.jsx)(F,{}),(0,s.jsx)(F,{})]}),(0,s.jsx)("h4",{"data-aos":"fade-up",children:"Single-cell Therapeutics AI"}),(0,s.jsx)("p",{"data-aos":"fade-up",children:"PyTDC presents datasets, tools, models, and benchmarks integrating cell-type-specific contextual features with ML tasks across therapeutics. We present four tasks for contextual learning at single-cell resolution."}),(0,s.jsx)("a",{"data-aos":"fade-up",className:"btn dream-btn mt-30",href:"/multi_pred_tasks/scdti",style:{borderColor:"#FFDF00"},children:"Single-cell Drug Target Nomination"})]})}),L=e=>{let{img:t}=e;return(0,s.jsx)("section",{className:"about-us-area section-padding-0-100 clearfix",id:"about",children:(0,s.jsx)("div",{className:"container",children:(0,s.jsxs)("div",{className:"row align-items-center",children:[(0,s.jsx)(E,{img:t}),(0,s.jsx)(S,{})]})})})},R=()=>(0,s.jsx)("span",{}),B=e=>{let{title:t,text:a,color1:i,color2:n}=e;return(0,s.jsxs)("div",{className:"section-heading text-center",children:[(0,s.jsx)("div",{className:"dream-dots justify-content-center","data-aos":"fade-up","data-aos-delay":"200",children:Array(7).fill(null).map((e,t)=>(0,s.jsx)(R,{},t))}),(0,s.jsx)("h2",{"data-aos":"fade-up","data-aos-delay":"300",style:{color:i||"white"},children:t}),(0,s.jsx)("p",{"data-aos":"fade-up","data-aos-delay":"400",style:{color:n||"white"},children:a})]})},H=e=>{let{timelineClass:t,title:a,date_from:i,date_to:n,content:l}=e;return(0,s.jsxs)("div",{className:t,children:[(0,s.jsx)("h3",{children:a}),(0,s.jsx)("span",{className:"date",children:i})," ",(0,s.jsx)("span",{className:"between",children:"to"})," ",(0,s.jsx)("span",{className:"date",children:n}),(0,s.jsx)("p",{children:l})]})},U=e=>{let{data:t}=e;return(0,s.jsxs)("section",{className:"roadmap",style:{paddingBottom:0},id:"roadmap",children:[(0,s.jsx)(B,{title:"PyTDC Roadmap",text:""}),(0,s.jsx)("div",{className:"container",children:(0,s.jsx)("div",{className:"row",children:(0,s.jsx)("div",{className:"timeline-split",children:(0,s.jsx)("div",{className:"col-lg-12 col-md-12 col-sm-12",children:(0,s.jsxs)("div",{className:"timeline section-box-margin",children:[t&&t.map((e,t)=>(0,s.jsx)(H,{timelineClass:e.timelineClass,title:e.title,date_from:e.date_from,date_to:e.date_to,content:e.content},t)),(0,s.jsx)("div",{className:"circle"})]})})})})})]})},W=e=>{let{title:t,content:a,link:i}=e;return(0,s.jsx)("div",{className:"col-12 col-sm-6 col-lg-4","data-aos":"fade-up",children:(0,s.jsxs)("div",{className:"service_single_content text-left mb-100",children:[(0,s.jsx)("h6",{onClick:()=>{window.open(i)},children:t}),(0,s.jsx)("p",{children:a})]})})},G=e=>{let{data:t}=e;return(0,s.jsx)("section",{className:"our_services_area section-padding-100-70 clearfix",id:"services",children:(0,s.jsxs)("div",{className:"container",children:[(0,s.jsx)(B,{title:"PyTDC ML Tasks",text:"We introduce therapeutics ML tasks, datasets, and benchmarks spanning an unprecedented number of modalities. Modalities in PyTDC include but are not limited to: single-cell gene expression atlases, single-cell chemical and genetic perturbations, clinical trial outcome data, peptide sequence data, peptidomimetics protein-peptide interaction data from AS-MS spectroscopy, novel 3D structural protein data, and cell-type-specific protein-protein interaction networks at single-cell resolution. PyTDC introduces ML tasks taking on open challenges, including the inferential gap in precision medicine and evaluation on longitudinal data, model generalization across cell lines and single-cell perturbations that were not encountered during  model training, and evaluation of models across a broad range of diverse biological contexts."}),(0,s.jsx)("div",{className:"row",children:t&&t.map((e,t)=>(0,s.jsx)(W,{title:e.title,content:e.content,link:e.link},t))})]})})},O=()=>(0,s.jsx)("section",{className:"partners",children:(0,s.jsx)(B,{title:"Our Collaborators",text:"To be disclosed"})}),z=()=>(0,s.jsx)("section",{className:"our_team_area section-padding-0-0 clearfix",id:"team",children:(0,s.jsx)("div",{className:"container",children:(0,s.jsx)("div",{className:"row",children:(0,s.jsx)("div",{className:"col-12",children:(0,s.jsx)(B,{title:"PyTDC Team",text:"To be disclosed",color1:"white",color2:"white"})})})})}),J=e=>{let{title:t,link:a}=e;return(0,s.jsx)("div",{className:"col-md-6 col-sm-6 col-xs-12","data-aos":"fade-up",children:(0,s.jsx)("a",{href:a,children:(0,s.jsx)("div",{className:"doc-element",children:(0,s.jsx)("a",{className:"document-entry",href:a,children:(0,s.jsx)("span",{className:"title",children:t})})})})})},V=e=>{let{data:t}=e;return(0,s.jsx)("div",{className:"ico-docs",children:(0,s.jsx)("div",{children:t&&t.map((e,t)=>(0,s.jsx)(J,{title:e.title,link:e.link},t))})})},X=()=>(0,s.jsxs)("div",{className:"welcome-content",children:[(0,s.jsx)("h1",{style:{color:"#3A9C94",fontFamily:"sans-serif"},children:"PyTDC-NextML"}),(0,s.jsx)("h1",{style:{color:"#FFB300",fontFamily:"sans-serif"},children:"A multimodal ML platform for biomedical foundation models"}),(0,s.jsx)("p",{}),(0,s.jsx)("h3",{style:{color:"#FFEFC2",fontFamily:"sans-serif"},children:"Learn more about our alpha release of PyTDC (scFMs) below!"}),(0,s.jsx)("p",{children:"We present PyTDC, a machine-learning platform providing streamlined training, evaluation, and inference software for biological foundation models to accelerate research in transfer learning method development in therapeutics."}),(0,s.jsxs)("div",{className:"row",children:[(0,s.jsx)("div",{className:"col-md-2"}),(0,s.jsx)("div",{className:"col-md-8",children:(0,s.jsx)(V,{data:p})}),(0,s.jsx)("div",{className:"col-md-2"})]}),(0,s.jsx)("p",{}),(0,s.jsx)("div",{className:"dream-btn-group",children:(0,s.jsx)("a",{href:"/home",className:"btn dream-btn",style:{borderColor:"#FFDF00",marginRight:"5px"},children:"Go to PyTDC homepage"})})]}),Q=()=>(0,s.jsx)("div",{className:"dream-blip blip1"}),q=()=>(0,s.jsxs)("section",{className:"welcome_area clearfix dzsparallaxer auto-init ico fullwidth","data-options":{direction:"normal"},id:"home",children:[(0,s.jsx)("div",{className:"divimage dzsparallaxer--target Home1WelcomeAreaIMG"}),(0,s.jsxs)("div",{className:"hero-content tdc-black",children:[[,,,,].fill(null).map((e,t)=>(0,s.jsx)(Q,{},t)),(0,s.jsx)("div",{className:"container",children:(0,s.jsx)(k(),{src:"../../../assets/img/Website Text.png",alt:"Website Text",layout:"fill"})}),(0,s.jsx)("div",{className:"container h-100",children:(0,s.jsxs)("div",{className:"row h-100 align-items-center",children:[(0,s.jsx)("div",{className:"col-12 col-lg-12 col-md-12",style:{textAlign:"center"},children:(0,s.jsx)(X,{})}),(0,s.jsx)("div",{className:"col-12 col-lg-6 col-md-12",children:(0,s.jsx)("div",{className:"main-ilustration"})})]})})]})]}),Y=e=>{let{nameIco:t,url:a}=e;return(0,s.jsx)("li",{children:(0,s.jsx)("a",{href:a,children:(0,s.jsx)("i",{className:t,"aria-hidden":"true"})})})},K=e=>{let{data:t}=e;return(0,s.jsx)("div",{className:"vertical-social",children:(0,s.jsx)("ul",{children:t&&t.map((e,t)=>(0,s.jsx)(Y,{nameIco:e.nameIco,url:e.link},t))})})},Z=()=>((0,i.useEffect)(()=>{x("PyTDC: A multimodal machine learning training, evaluation, and inference platform for biomedical foundation models")},[]),(0,i.useEffect)(()=>{if("undefined"!=typeof document){let e=document.getElementsByTagName("body")[0];"PyTDC: A multimodal machine learning training, evaluation, and inference platform for biomedical foundation models"===document.title?e.style.backgroundImage="linear-gradient(180deg,#240044 0,#0f0240 25%,#400959 40%,#0f0240 65%,#0f0240)":e.style.backgroundImage="linear-gradient(to right, #4834d4, #341f97)"}},[]),(0,s.jsxs)(s.Fragment,{children:[(0,s.jsx)(N,{}),(0,s.jsxs)("div",{className:"HomeDemo1",children:[(0,s.jsx)(q,{}),(0,s.jsx)(K,{data:m}),(0,s.jsx)(P,{img:r.src}),(0,s.jsx)(D,{img:d.src}),(0,s.jsx)(L,{img:c.src}),(0,s.jsx)(G,{data:h}),(0,s.jsx)(U,{data:u}),(0,s.jsx)(z,{data:g}),(0,s.jsx)(O,{data:f})]})]}));a(3173),a(8736);let $=()=>((0,i.useEffect)(()=>{o().init({duration:1e3})},[]),(0,s.jsxs)("div",{className:"App",children:[(0,s.jsxs)(n.m,{children:[(0,s.jsx)("meta",{charset:"utf-8"}),(0,s.jsx)("meta",{name:"viewport",content:"width=device-width, initial-scale=1, shrink-to-fit=no"}),(0,s.jsx)("title",{children:"PyTDC: A multimodal ML platform for biomedical foundation models."}),(0,s.jsx)("link",{href:"https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700",rel:"stylesheet"}),(0,s.jsx)("link",{rel:"stylesheet",href:"https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css",integrity:"sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=",crossorigin:"anonymous"})]}),(0,s.jsx)(Z,{})]}))}}]);