{% extends "index_template.html" %}
{% block content %}
    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-4-desktop is-4-tablet">
                    

<aside class="menu">

    <p class="menu-label"></p>
    <ul class="menu-list">
        
        <li>
            <a href="/multi_pred_tasks/overview" class=""><strong>Multi-instance Prediction Problem</strong></a>
            
            <ul>
                
                
                <li><a href="/multi_pred_tasks/dti" class=""><code>DTI</code> Drug-Target Interaction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/ddi" class=""><code>DDI</code> Drug-Drug Interaction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/ppi" class=""><code>PPI</code> Protein-Protein Interaction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/gdi" class=""><code>GDA</code> Gene-Disease Association</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/drugres" class=""><code>DrugRes</code> Drug Response Prediction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/drugsyn" class=""><code>DrugSyn</code> Drug Synergy Prediction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/peptidemhc" class=""><code>PeptideMHC</code> Peptide-MHC Binding</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/antibodyaff" class=""><code>AntibodyAff</code> Antibody-Antigen Affinity</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/mti" class=""><code>MTI</code> miRNA-Target Interaction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/catalyst" class=""><code>Catalyst</code> Catalyst Prediction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/tcrepitope" class=""><code>TCREpitopeBinding</code> TCR-Epitope Binding</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/trialoutcome" class=""><code>TrialOutcome</code> Clinical Trial Outcome Prediction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/proteinpeptide" class=""><code>ProteinPeptide</code> Protein-Peptide Interaction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/counterfactual" class=""><code>PerturbOutcome</code> Counterfactual Prediction</a></li>
                
                
                
                <li><a href="/multi_pred_tasks/scdti" class=""><code>scDTI</code> Single-cell Drug-Target Interaction</a></li>
                
                
            </ul>
            
        </li>
            
    </ul>

    <p class="menu-label"></p>
    <ul class="menu-list">
        
        <li>
            <a href="/overview" class=""><strong>Others</strong></a>
            
            <ul>
                
                
                <li><a href="/single_pred_tasks/overview" class="">Single-instance Prediction Problem</a></li>
                
                
                
                <li><a href="/generation_tasks/overview" class="">Generation Problem</a></li>
                
                
            </ul>
            
        </li>
            
    </ul>

</aside>
                </div>
                
                <div class="column is-8">
                    
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Dataset Index</p>
        <ul class="menu-list">
  <li><a href="#ye-x-et-al">Ye X et al</a></li>
</ul>
    </div>
</div>




<div class="content">
    <h1 id="protein-peptide-interaction-prediction-task-overview">Protein-Peptide Interaction Prediction Task Overview</h1>

<div class="box">
	

<p class="is-size-6">  <strong> Definition: </strong> 
Protein-peptide interactions are critical for many biological processes and have significant implications in research and therapeutic development. Peptides and protein-peptide interactions play crucial roles in drug discovery for several reasons, including their involvement in biological processes, their therapeutic potential, and their ability to modulate difficult targets like protein-protein interactions. <br /> Protein-peptide interactions are essential in regulating various cellular functions such as signal transduction, protein trafficking, and epigenetic regulation. For instance, peptides derived from histone proteins play crucial roles in chromatin structure and gene expression regulation (Plach et al., 2017). <br /> Peptides can serve as inhibitors or modulators of protein-protein interactions, offering a potential avenue for drug development, especially for targets that are challenging for small molecules (Kilburg &amp; Gallicchio, 2016). <br /> Peptides and peptide derivatives are increasingly being recognized for their potential as therapeutic agents to target protein-protein interactions, which are often implicated in diseases like cancer (Nevola &amp; Giralt, 2015). <br /> Studying protein-peptide interactions provides insights into the structural basis of these interactions, which can be leveraged to design better therapeutic molecules (London et al., 2010).
<br />
<br />
Despite the availability of several benchmarks for protein-protein interactions, there is a comparatively large gap compared to benchmarks for protein-peptide binding affinity prediction. For example, the renowned multi-task benchmark for Protein sEquence undERstanding (PEER) (Xu, Minghao et al., 2022), lacks support for a specifically defined protein-peptide binding affinity prediction task. Protein-peptide binding affinity prediction and protein-protein binding affinity prediction involve similar underlying biological interactions, but they differ significantly in their complexity and the methods used to predict them (Abdin, Osana et al., 2022).
</p>

<p class="is-size-6"> <strong> Impact: </strong>  
Machine learning models can rapidly predict binding affinities between proteins and peptides, which is essential for identifying potential therapeutic peptides that can modulate protein functions. This accelerates the drug discovery process by reducing the need for extensive experimental assays (Li et al., 2019). <br /> Traditional methods of determining protein-peptide binding affinities, such as experimental binding assays, are expensive and time-consuming. Machine learning offers a cost-effective alternative by utilizing existing data to predict new interactions without the need for additional wet-lab experiments (Kundu et al., 2018). <br /> Protein-peptide interactions are complex and involve various biophysical and biochemical properties. Machine learning models can handle this complexity by integrating multiple data types (e.g., sequence, structural, physicochemical properties) to make accurate predictions (Aranha et al., 2020). <br /> Predicting protein-peptide binding affinities is vital for developing personalized medicine approaches, such as designing peptide-based vaccines and immunotherapies. Accurate predictions help in selecting peptides that bind strongly to specific proteins, enhancing the effectiveness of treatments (Bhattacharya et al., 2017).
</p>

<p class="is-size-6"> <strong> Generalization: </strong> 
There is generally less experimental data available for protein-peptide interactions compared to protein-protein interactions, which hampers the development and validation of predictive models (Chang &amp; Perez, 2022). Capturing the dynamic nature of protein-peptide interactions requires advanced sampling techniques and computational resources, making the prediction of binding affinities more complex (Antes et al., 2014). TDC-2 provides benchmarks integrating newly discovered peptides to test ML models on their ability to generalize to cutting-edge peptidomimetics.
</p>

<p class="is-size-6"> <strong> Product: </strong>  biopharmaceuticals</p>

<p class="is-size-6"> <strong> Pipeline: </strong>  Target discovery, Lead Optimization. </p>

</div>

<h3 id="ye-x-et-al">Ye X et al</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Affinity selection-mass spectrometry data of discovered ligands against single biomolecular targets (MDM2, ACE2, 12ca5) from the Pentelute Lab of MIT This dataset contains affinity selection-mass spectrometry data of discovered ligands against single biomolecular targets. Several of these AS-MS discovered ligands were taken forward for experimental validation to determine the binding affinity (KD) as measured by biolayer interferometry (BLI) to the listed target protein. If listed as a "putative binder," AS-MS alone was used to isolate the ligands to the target, with KD &lt; 1 uM required and often observed in orthogonal assays, though there is some (&lt; 50%) chance that the ligand is nonspecific. Most of the ligands are putative binders with 4446 total provided. For those characterized by BLI (only 34 total), the average KD is 266 Â± 44 nM, median KD is 9.4 nM. 
<p class="is-size-6">  <strong> Task Description: </strong>Binary Classification. Given the target amino acid sequence pairs, predict if they interact or not. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 34 positive ligands, 4446 putative binders, 3 proteins </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split (stratified)</span> </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.multi_pred</span> <span class="kn">import</span> <span class="n">ProteinPeptide</span> 
<span class="n">data</span> <span class="o">=</span> <span class="n">ProteinPeptide</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'brown_mdm2_ace2_12ca5'</span><span class="p">,</span> <span class="n">path</span> <span class="o">=</span> <span class="s">'./data'</span><span class="p">)</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6"> <strong>Note:</strong> If listed as a "putative binder," AS-MS alone was used to isolate the ligands to the target, with KD &lt; 1 uM required and often observed in orthogonal assays, though there is some (&lt; 50%) chance that the ligand is nonspecific. </p>

<p class="is-size-6"> <strong>Note:</strong> Only 10% of the data is provided in a train set for fine-tuning on the benchmark associated to this dataset with the remaining 90% used as the test set. Outside of the benchmark, the user may define splits as desired. </p>

<p class="is-size-6">  <strong> References: </strong>  </p>

<a href="https://doi.org/10.1038/s42004-022-00737-w">[1] Ye X, Lee YC, Gates ZP, Ling Y, Mortensen JC, Yang FS, Lin YS, Pentelute BL. Binary combinatorial scanning reveals potent poly-alanine-substituted inhibitors of protein-protein interactions. Commun Chem. 2022 Oct 14;5(1):128. doi: 10.1038/s42004-022-00737-w. PMID: 36697672; PMCID: PMC9814900. </a> 

<p class="is-size-6"> <strong> Dataset License: </strong><a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>


<hr />
</p>

</div>
                </div>
                
            </div>
        </div>
    </section>
    
 {% endblock %}